AmplifyTests/CategoryTests/Auth/AuthCategoryConfigurationTests.swift

```diff
@@ -0,0 +1,367 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import XCTest
+import CwlPreconditionTesting
+
+@testable import AmplifyTestCommon
+@testable import Amplify
+
+class AuthCategoryConfigurationTests: XCTestCase {
```

Comment:
For the names of the individual test methods:

The class is testing the `AuthCategoryConfiguration`. However, the tests  are named like they're testing a plugin. They should probably instead be called like `testCanConfigureCategory`.

Reply:
Changed names